services/matrix/matrix.service.js

```diff
@@ -76,6 +85,15 @@ export default class Matrix extends BaseJsonService {
             name: 'server_fqdn',
             example: 'matrix.org',
           }),
+          queryParam({
+            name: 'fetchMode',
+            example: 'guest',
+            description: `If not specified, the default fetch mode is <code>guest</code> (except for matrix.org).`,
```

Comment:
I think its reasonable to do most of the explaining in the docs at the top of the page, but in the param description here could we summarise that "guest" means guest auth and "summary" means use the experimental summary endpoint as well as documenting the default.

Reply:
Done 29481c975f85e71a82b4ae94415c5f1fb82cb0c8